activities/Abecedarium.activity/buttons.js

```diff
@@ -40,6 +40,31 @@ enyo.kind({
 	// Constructor
 	rendered: function() {
 		this.inherited(arguments);
+		function hideLangandCasebutton(){
```

Comment:
I'm not a big fan of the idea to do a mix of Lang and Case in a method attached to render the Case object.
It will be better to split this in two methods, one for Case in the Case rendering and one for Lang in the Lang rendering.